src/JustSaying/Messaging/Middleware/ErrorHandling/ErrorHandlerMiddleware.cs

```diff
@@ -22,6 +22,11 @@ protected override async Task<bool> RunInnerAsync(HandleMessageContext context,
         {
             return await func(stoppingToken).ConfigureAwait(false);
         }
+        catch (OperationCanceledException)
```

Comment:
Could this potentially catch timeouts on the SNS/SQS side and prevent logging them? If so is this desired?

Reply:
We should be good, `SqsPostProcessorMiddleware` is typically after this middleware when using the baked in middleware pipelines, so we shouldn't catch SQS timeouts. Polling happens elsewhere too.